src/main/java/racingcar/exception/InputRoundValidator.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.exception;
+
+import static racingcar.constant.ErrorMessage.*;
+
+public class InputRoundValidator {
+    public static final int TRIAL_TIME_MIN = 0;
+    public static void validateRound(String input) { // 시도 횟수 유효성 검사
+        if (input == null || input.isEmpty()) { // 입력이 null이거나 빈 문자열인 경우
+            throw new IllegalArgumentException(INPUT_BLANK);
+        }
+
+        for (char c : input.toCharArray()) { // 입력값이 숫자가 아닌 문자를 포함하는 경우
+            if (!Character.isDigit(c)) {
+                throw new IllegalArgumentException(INPUT_NOT_INTEGER);
+            }
+        }
```

Comment:
아래 두 가지 방법도 고려해보면 좋을 거 같습니다

1. 정수로 변환 과정 중 예외 발생 시 예외 던지기
```suggestion
try {
    Integer.parseInt(input);
} catch (NumberFormatException e) {
    throw new IllegalArgumentException(INPUT_NOT_INTEGER);
}
```

2. 정규표현식 사용
- 이 부분은 저도 아직 미숙하지만 간결하며 강력하기에 익혀두시기를 추천드립니다
```suggestion
if (!input.matches("\\d+")) {
    throw new IllegalArgumentException(INPUT_NOT_INTEGER);
}
```

Reply:
정규표현식.. 어렵지만 진짜 강력한 것 같습니다!

이번 주차때는 한 번 연습해봐야겠네요

꼼꼼한 리뷰 감사합니다.